packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_control_pane.dart

```diff
@@ -15,24 +16,42 @@ class SnapshotControlPane extends StatelessWidget {
       : super(key: key);
 
   final DiffPaneController controller;
+  static const _classFilterWidth = 200.0;
 
   @override
   Widget build(BuildContext context) {
     return ValueListenableBuilder<bool>(
       valueListenable: controller.isProcessing,
       builder: (_, isProcessing, __) {
-        final current = controller.selectedItem as SnapshotListItem;
+        final current = controller.selectedSnapshotItem as SnapshotInstanceItem;
 
         return Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
           children: [
-            const SizedBox(width: defaultSpacing),
-            if (!isProcessing && current.heap != null) ...[
-              _DiffDropdown(
-                current: current,
-                list: controller.snapshots,
-              ),
-              const SizedBox(width: defaultSpacing),
-            ],
+            // This child is aligned to the left.
```

Comment:
nit: these comments shouldn't be necessary since `MainAxisAlignment.spaceBetween` implies this